### PR TITLE
Creating expandable  card for concert attire

### PR DESCRIPTION
**Pull request by apatyk**

### CODE REVIEW

File: src/templates/ensembles.js [hunk 1]
```diff
@@ -17,7 +16,7 @@ export const EnsemblesTemplate = ({
   header: { topText, bottomText },
   subheading,
   about,
-  button: { buttonText, buttonLink },
+  concertAttire: { attireHeading, attireContent, attireDrawer  },
```
Comment (GusTheBusNG) [inline src/templates/ensembles.js:19]: If we de-structure the elements here then if they don't have them the admin panel throws an error. I suggest adding in some default props and / or the default property on config.yaml

File: src/templates/ensembles.js [hunk 2]
```diff
@@ -30,9 +29,12 @@ export const EnsemblesTemplate = ({
     <div className="ensembles__content">
       <Header topText={topText} bottomText={bottomText} />
       <TextContent header={subheading} content={about} />
-      <Button link={buttonLink}>
-        {buttonText}
-      </Button>
+      <FloatingCard
```
Comment (GusTheBusNG) [inline src/templates/ensembles.js:32]: Screen Shot 2020-04-04 at 10 08 12 PM
The space at the bottom seems a little much without the email icon

File: static/admin/config.yml [hunk 3]
```diff
@@ -26,10 +26,13 @@ collections:
...
-          {label: "Button Label", name: buttonText, widget: string},
-          {label: "Button Link", name: buttonLink, widget: string}
-        ]}
+      - label: "Concert Attire"
+        name: concertAttire
+        widget: object
+        fields:
+          - {label: "Concert Attire Heading", name: attireHeading, widget: string, default: 'Concert Attire'}
+          - {label: "Concert Attire Description", name: attireContent, widget: text, default: 'Brief description...'}
+          - {label: "Concert Attire Content", name: attireDrawer, widget: text, default: 'More text here...'}
```
Comment (apatyk) [inline static/admin/config.yml:35]: @GusTheBusNG Does this fix the issue?

Comment (GusTheBusNG) [reply static/admin/config.yml:35]: Yeah I think so, I'll more into it on the `bullet-proof` branch

**Review by GusTheBusNG — APPROVED** [review]

**Pull request merged**